src/components/home/style.scss

```diff
@@ -0,0 +1,104 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+.container {
+  background-color: #f0f0f0;
+  padding: 10%;
+}
+
+.title {
+  font-family: "Lato", sans-serif;
+  position: relative;
+  font-size: 39px;
+  color: #2c3648;
+  margin-bottom: 5%;
+  margin-left: 1%;
+  text-align: center;
+  max-width: 300px;
+  &::before {
+    content: "";
+    display: block;
+    width: 50px;
+    height: 3px;
+    background-color: #2c3648;
+    position: absolute;
+    left: 0;
+    top: 50%;
+  }
+}
+.card {
+  width: 90%;
+  height: 200px;
+  position: relative;
+  margin: 10%;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  img {
+    height: 100%;
+    width: 100%;
+    object-fit: cover !important;
+    position: relative;
+  }
+  &::before {
+    content: "";
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: linear-gradient(
+        0deg,
+        rgba(44, 53, 71, 0.25),
+        rgba(44, 53, 71, 0.25)
+      ),
+      rgba(44, 53, 71, 0.25);
+    z-index: 1;
+  }
+}
+.cardTitle {
+  position: absolute;
+  bottom: 20px;
+  left: 0px;
+  padding: 3% 12%;
+  background-color: #fff;
+  font-family: "Lato";
+  font-size: 10px;
+  font-weight: 400;
+  z-index: 2;
+}
+.slick-dots {
+  background-color: #f0f0f0;
+  li {
+    button {
+      &::before {
+        font-size: 17px;
+        transform: matrix(0, 1, 1, 0, 0, 0);
+      }
+    }
+  }
+}
+.slick-slide.slick-center {
```

Comment:
add a comment above these that these are classes from react-slick, i'm assuming (since they're not camelcase). this will help people who look at your app later and your teammates know what these rules are. do it for all slick rules